introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js

```diff
@@ -0,0 +1,130 @@
+const $body = document.querySelector('body');
+const $buttonAddInput = document.querySelector('#add-input-button');
```

Comment:
I think it's not necessary to refer to the HTML element ``buttonAddInput`` => ``buttonAdd``. What do you think?

Reply:
Yes!, it looks much better that way. 
I also removed the following variable,
from $buttonRemoveInput to $buttonRemove :D